share/completions/ebuild.fish

```diff
@@ -12,6 +12,7 @@ complete -c ebuild -l skip-manifest	-d "Skip all manifest checks"
 
 ## Subcommands
 complete -c ebuild -n '__fish_seen_ebuild_arg' -xa 'help'        -d "Show help"
+complete -c ebuild -n '__fish_seen_ebuild_arg' -xa 'pretend'     -d "Run pkg_pretend()"
```

Comment:
What's a `pkg_pretend()`? Is it edible?

Reply:
I'm assuming this is a "--dry-run" option, that just pretends to do the operation?

Reply:
Yes, kinda: https://devmanual.gentoo.org/ebuild-writing/functions/pkg_pretend/index.html